.github/workflows/test.yml

```diff
@@ -0,0 +1,47 @@
+name: test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: "0 0 * * *" # JST 09:00
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container: ${{ matrix.ruby }}
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - ruby:2.6
```

Comment:
The min Ruby version is the least supported version.
ref: https://endoflife.date/ruby